src/_locales/fr/messages.json

```diff
@@ -1,5 +1,5 @@
 {
-    // manifest.json
+    // manifest.json 
```

Comment:
ehm, this is not indentation.

This is a nice "error" you spotted, but I actually meant to use 2 spaces instead of 4 spaces for the JSON…

Reply:
Well, I thought changing indentation in Notepad++ would be fine, but ... no.
this change was propably a mis-input ^^''

Reply:
Ah sorry, now actually saw it. Was too fast :smile: